network/index.md

```diff
@@ -1,103 +1,136 @@
-
-- [Standalone networking tutorial](network-tutorial-standalone.md)
-- [Host networking tutorial](network-tutorial-host.md)
-- [Overlay networking tutorial](network-tutorial-overlay.md)
-- [Macvlan networking tutorial](network-tutorial-macvlan.md)
+Container networking refers to the ability for containers to connect to and
+communicate with each other, or to non-Docker workloads.
+
+A container has no information about what kind of network it's attached to,
+or whether their peers are also Docker workloads or not.
+A container only sees a network interface with an IP address,
+a gateway, a routing table, DNS services, and other networking details.
+That is, unless the container uses the `none` network driver.
+
+This page describes networking from the point of view of the container,
+and the concepts around container networking.
+This page doesn't describe OS-specific details about how Docker networks work.
+For information about how Docker manipulates `iptables` rules on Linux,
+see [Packet filtering and firewalls](packet-filtering-firewalls.md).
+
+## Published ports
+
+By default, when you create or run a container using `docker create` or `docker run`,
+the container doesn't expose any of its ports to the outside world.
+Use the `--publish` or `-p` flag to make a port available to services
+outside of Docker.
+This creates a firewall rule in the host,
+mapping a container port to a port on the Docker host to the outside world.
+Here are some examples:
+
+| Flag value                      | Description                                                                                                                                           |
+| ------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `-p 8080:80`                    | Map TCP port 80 in the container to port `8080` on the Docker host.                                                                                   |
+| `-p 192.168.1.100:8080:80`      | Map TCP port 80 in the container to port `8080` on the Docker host for connections to host IP `192.168.1.100`.                                        |
+| `-p 8080:80/udp`                | Map UDP port 80 in the container to port `8080` on the Docker host.                                                                                   |
+| `-p 8080:80/tcp -p 8080:80/udp` | Map TCP port 80 in the container to TCP port `8080` on the Docker host, and map UDP port `80` in the container to UDP port `8080` on the Docker host. |
+
+> **Important**
+>
+> Publishing container ports is insecure by default. Meaning, when you publish
+> a container's ports it becomes available not only to the Docker host, but to
+> the outside world as well.
+>
+> To publish a container's port and only expose it to the Docker host, include
+> the localhost IP address (`127.0.0.1`) in the port mapping command.
```

Comment:
Presently, this is still reachable via connected LAN hosts and potentially to the published container IP + port: https://github.com/moby/moby/issues/45610#issuecomment-1562191303

Not sure if you want to raise awareness of that risk, but might make sense to if a fix will take some time to address.

Reply:
added: https://github.com/docker/docs/pull/17176/commits/84c5b318f5e600d649b9954950305c48ef07ffe5